python/tests/visualization/test_bloch_sphere_visualization.py

```diff
@@ -0,0 +1,131 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+import os 
+import pytest
+import numpy as np
+
+import cudaq
+import qutip
```

Comment:
Skip these tests if qutip module not available.
`qutip = pytest.importorskip("qutip")` before `import qutip`

Reply:
Fixed in [a3964e2](https://github.com/NVIDIA/cuda-quantum/pull/1763/commits/a3964e284910f99cbbddddfa7493747ab9735855).